benchmarking/chain-sync/analyse-logs.sh

```diff
@@ -1,10 +1,19 @@
 #!/bin/sh
+set -x
+logdir="$1"
+if test ! -d "${logdir}"
+then echo "ERROR:  log directory not found at '${logdir}'" >&2; exit 1; fi
 
 # get first block copy time
-LOGFILE=`ls -1 state-node-mainnet/node-0-*.log | head -3`
```

Comment:
the first reported slot might not be recorded in the first log file. that's why the script looks at the first three files.